scripts/commands/initialize.py

```diff
@@ -14,7 +14,9 @@
     CONFIGURATION_FILE,
     EnvironmentException,
     find_typeshed,
-    log)
+    log
```

Comment:
Comma

Reply:
Thanks, fixed.

Reply:
@dark I couldn't figure out how to run the python tests. Are there docs anywhere that talk about that?

Reply:
We run the tests internally with buck. I suspect with the current setup this wont work on github. As far as I know the integration we rely on is not quite open sourced yet I'll defer to @ambv on that one.